src/class/RacingGameViewer.js

```diff
@@ -0,0 +1,57 @@
+import * as readline from "node:readline/promises";
+
+const CAR_NAME_INPUT_GUIDE =
+  "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).\n";
+
+const RACING_ROUND_INPUT_GUIDE = "시도할 회수는 몇회인가요?\n";
+
+const RACING_SCORE_CHAR = "-";
+
+const WINNER_ANNOUNCEMENT_MESSAGE = "가 최종 우승했습니다.";
+
+const ROUND_HEADER_MESSAGE = "\n실행결과";
+
+export default class RacingGameViewer {
+  constructor() {
+    this.readline = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+  }
+
+  async getUserInput(question) {
+    return await this.readline.question(question);
+  }
+
+  async getRoundNumberInput() {
+    return await this.getUserInput(RACING_ROUND_INPUT_GUIDE);
+  }
+
+  async getCarNamesInput() {
+    return await this.getUserInput(CAR_NAME_INPUT_GUIDE);
+  }
```

Comment:
```suggestion
  getUserInput(question) {
    return this.readline.question(question);
  }

  getRoundNumberInput() {
    return this.getUserInput(RACING_ROUND_INPUT_GUIDE);
  }

  getCarNamesInput() {
    return this.getUserInput(CAR_NAME_INPUT_GUIDE);
  }
```

async/await은 제거해도 무방해보이네요 ㅎㅎ

그렇다면 이 상황에서 붙이고 안 붙이고가 어떤 차이가 있을까요?
동작이 완전히 똑같진 않아서요!